crates/turborepo-filewatch/src/cookie_jar.rs

```diff
@@ -0,0 +1,341 @@
+use std::{
+    collections::HashMap,
+    fs::OpenOptions,
+    path::PathBuf,
+    sync::{atomic::AtomicUsize, Arc},
+    time::Duration,
+};
+
+use notify::{Event, EventKind};
+use thiserror::Error;
+use tokio::{
+    sync::{broadcast, oneshot, Mutex},
```

Comment:
I think we should be using the standard mutex here.

[tokio mutex vs std mutex](https://tokio.rs/tokio/tutorial/shared-state#on-using-stdsyncmutex) and the [alternatives suggested](https://tokio.rs/tokio/tutorial/shared-state#holding-a-mutexguard-across-an-await)

TL;DR use std unless you need to hold the mutex across an await and even then, you should try to avoid doing that

Reply:
Good reference. Updated to `std::sync::Mutex`